diff --git a/lib/module/index.js b/lib/module/index.js
index da49dc1c87e91bc09060f2256f1693fc15bf79d9..d7d598509acc1a61e33ef78eef7f961bb1a913b8 100644
--- a/lib/module/index.js
+++ b/lib/module/index.js
@@ -1,6 +1,6 @@
 "use strict";
 
-export * from './Mapbox';
-import * as Mapbox from './Mapbox';
+export * from './Mapbox.native.js';
+import * as Mapbox from './Mapbox.native.js';
 export default Mapbox;
 //# sourceMappingURL=index.js.map
