diff --git a/ios/DevMenuViewController.swift b/ios/DevMenuViewController.swift
index c5e7494101dc0d0b4292b0f6f8b282efa8910696..0b48d9f965d438f9e12169879bc18c9ed09ca3fa 100644
--- a/ios/DevMenuViewController.swift
+++ b/ios/DevMenuViewController.swift
@@ -61,7 +61,13 @@ class DevMenuViewController: UIViewController {
   // MARK: private
 
   private func initialProps() -> [String: Any] {
-    let isSimulator = TARGET_IPHONE_SIMULATOR > 0
+    let isSimulator: Bool = {
+      #if targetEnvironment(simulator)
+        return true
+      #else
+        return false
+      #endif
+    }()
     
     return [
       "showOnboardingView": manager.shouldShowOnboarding(),
